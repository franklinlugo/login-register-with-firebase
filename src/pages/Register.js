import React from 'react';
import { FormLayout } from 'components';

function Register() {
  return <FormLayout>Register</FormLayout>;
}

export default Register;
