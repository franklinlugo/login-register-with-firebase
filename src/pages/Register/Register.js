import React from 'react';
import { FormLayout, RegisterForm } from 'components';

function Register() {
  return (
    <FormLayout>
      <RegisterForm />
    </FormLayout>
  );
}

export default Register;
