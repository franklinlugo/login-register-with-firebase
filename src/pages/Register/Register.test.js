import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Register from './Register';

test('Register render', () => {
  const { asFragment } = render(
    <Router>
      <Register />
    </Router>,
  );
  expect(asFragment()).toMatchSnapshot();
});
