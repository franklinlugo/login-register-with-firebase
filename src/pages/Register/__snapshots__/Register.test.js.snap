// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register render 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm lnfnfY"
  >
    <div
      class="sc-fzqBZW dgHQfD"
    >
      <h3
        class="sc-fzqNJr dsyDpg"
      >
        REGISTER
      </h3>
      <form
        novalidate=""
      >
        <div
          class="sc-AxmLO cwHqHy"
        >
          <div
            class="sc-AxheI fsezoq"
          >
            <label
              class="sc-AxiKw jrbXxa"
              for="email"
            >
              Email
            </label>
            <input
              class="sc-AxirZ gthKbY"
              id="email"
              name="email"
              type="email"
              value=""
            />
            <div
              class="sc-AxhCb jGNELg"
            >
              <svg
                viewBox="0 0 512 512"
                width="18px"
              >
                <path
                  d="M485.743 85.333H26.257C11.815 85.333 0 97.148 0 111.589V400.41c0 14.44 11.815 26.257 26.257 26.257h459.487c14.44 0 26.257-11.815 26.257-26.257V111.589c-.001-14.441-11.816-26.256-26.258-26.256zm-9.853 19.691L271.104 258.626c-3.682 2.802-9.334 4.555-15.105 4.529-5.77.026-11.421-1.727-15.104-4.529L36.109 105.024H475.89zM366.5 268.761l111.59 137.847c.112.138.249.243.368.368H33.542c.118-.131.256-.23.368-.368L145.5 268.761a9.846 9.846 0 00-1.464-13.851c-4.227-3.419-10.424-2.771-13.844 1.457l-110.5 136.501V117.332l209.394 157.046c7.871 5.862 17.447 8.442 26.912 8.468 9.452-.02 19.036-2.6 26.912-8.468l209.394-157.046v275.534L381.807 256.367c-3.42-4.227-9.623-4.877-13.844-1.457-4.234 3.419-4.884 9.624-1.463 13.851z"
                />
              </svg>
            </div>
            <div
              class="sc-AxgMl hGREOL"
            />
          </div>
        </div>
        <div
          class="sc-AxmLO cwHqHy"
        >
          <div
            class="sc-AxheI fsezoq"
          >
            <label
              class="sc-AxiKw jrbXxa"
              for="email"
            >
              Password
            </label>
            <input
              class="sc-AxirZ gthKbY"
              id="email"
              name="password"
              type="password"
              value=""
            />
            <div
              class="sc-AxhCb jGNELg"
            >
              <svg
                viewBox="0 0 512 512"
                width="18px"
              >
                <path
                  d="M450.462 194.372h-39.028v-38.179C411.428 69.929 341.739.007 256.001 0c-85.738.007-155.628 69.929-155.635 156.192v38.179H61.538c-14.26 0-25.928 11.74-25.928 26.09V485.91c0 14.351 11.667 26.09 25.928 26.09h388.925c14.26 0 25.928-11.74 25.928-26.09V220.462c0-14.352-11.668-26.09-25.929-26.09zm-330.65-38.18c.007-37.758 15.583-71.853 40.17-96.609 24.607-24.736 58.488-40.008 96.02-40.014 37.525.007 71.413 15.278 96.013 40.014 24.593 24.756 39.968 58.851 39.975 96.609v38.179H119.812v-38.179zm330.652 336.239H61.538c-3.513 0-6.482-2.989-6.482-6.521V220.462c0-3.533 2.969-6.521 6.482-6.521h388.925c3.513 0 6.482 2.982 6.482 6.521V485.91h.001c-.001 3.532-2.97 6.521-6.482 6.521z"
                />
                <path
                  d="M256.001 296.763c-25.067 0-45.375 20.438-45.375 45.654 0 21.858 15.259 40.098 35.652 44.577v46.73c0 5.406 4.356 9.782 9.723 9.782 5.367 0 9.723-4.376 9.723-9.782v-46.73c20.392-4.48 35.652-22.727 35.652-44.577-.001-25.211-20.309-45.654-45.375-45.654zm0 71.749c-14.3 0-25.928-11.707-25.928-26.09s11.629-26.09 25.928-26.09c14.3 0 25.928 11.707 25.928 26.09s-11.629 26.09-25.928 26.09z"
                />
              </svg>
            </div>
            <span
              class="sc-AxhUy bqvrtG"
              role="button"
            >
              <svg
                height="17.999"
                viewBox="0 0 24 17.999"
                width="24"
              >
                <path
                  d="M14.828 6.171a.5.5 0 00-.707.707A2.98 2.98 0 0115 9c0 1.654-1.346 3-3 3a2.976 2.976 0 01-2.121-.879.5.5 0 00-.707.707A3.968 3.968 0 0012 13c2.206 0 4-1.794 4-4a3.968 3.968 0 00-1.172-2.828zm-2.131-1.103A4.008 4.008 0 0012 5C9.794 5 8 6.793 8 9c0 .238.03.47.07.697a.5.5 0 00.984-.174A2.9 2.9 0 019 9c0-1.654 1.346-3 3-3 .18 0 .353.023.523.053a.499.499 0 10.174-.984z"
                />
                <path
                  d="M23.881 8.676c-.113-.134-2.818-3.29-6.549-5.224a.5.5 0 00-.459.89c2.874 1.487 5.17 3.81 5.955 4.657-1.192 1.291-5.883 6-10.828 6-1.678 0-3.369-.405-5.028-1.205a.499.499 0 10-.434.9C8.333 15.561 10.171 16 12 16c6.161 0 11.651-6.404 11.882-6.677a.5.5 0 00-.001-.646zm-8.6-6.101C14.143 2.192 13.039 2 12 2 5.839 2 .35 8.403.118 8.676a.5.5 0 00-.028.61c.07.098 1.722 2.445 4.587 4.348a.495.495 0 00.692-.141.499.499 0 00-.14-.692 18.286 18.286 0 01-4.083-3.775C2.297 7.776 7.02 3 12 3c.931 0 1.928.176 2.962.523a.493.493 0 00.633-.314.5.5 0 00-.314-.633z"
                />
                <path
                  d="M20.853.146a.5.5 0 00-.707 0l-17 17a.5.5 0 00.707.707l17-17a.5.5 0 000-.707z"
                />
              </svg>
            </span>
            <div
              class="sc-AxgMl hGREOL"
            />
          </div>
        </div>
        <button
          class="sc-fzozJi kTvlDf"
          type="submit"
        >
          LOGIN
        </button>
      </form>
      <p
        class="sc-fzoyAV eopbGX"
      >
        Have an account? 
        <a
          href="/login"
        >
          Log in
        </a>
      </p>
    </div>
  </div>
</DocumentFragment>
`;
